src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -152,9 +152,9 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
   --pf-c-dropdown__group-title--PaddingRight: var(--pf-c-dropdown__menu-item--PaddingRight);
   --pf-c-dropdown__group-title--PaddingBottom: var(--pf-c-dropdown__menu-item--PaddingBottom);
   --pf-c-dropdown__group-title--PaddingLeft: var(--pf-c-dropdown__menu-item--PaddingLeft);
-  --pf-c-dropdown__group-title--FontSize: var(--pf-global--FontSize--sm);
-  --pf-c-dropdown__group-title--FontWeight: var(--pf-global--FontWeight--semi-bold);
-  --pf-c-dropdown__group-title--Color: var(--pf-global--Color--dark-200);
+  --pf-c-dropdown__group-title--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-dropdown__group-title--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-dropdown__group-title--Color: var(--pf-global--Color--200);
```

Comment:
This should stay as `--dark-200` - it has to do with this legacy "dark" theme that some components use. The dropdown menu should always remain a light box, even when it's in this legacy dark theme and the `--dark-200` color retains the color where as `--200` will change in the legacy dark theme, and we don't want that for contents of the dropdown menu.

```suggestion
  --pf-c-dropdown__group-title--Color: var(--pf-global--Color--dark-200);
```

Here's what happens with this change - note I added `.pf-t-dark` to the example box to trigger the legacy dark theme

<img width="943" alt="Screen Shot 2022-07-19 at 4 38 26 PM" src="https://user-images.githubusercontent.com/35148959/179853402-dd1a3c29-aecc-4a28-a4d5-5047f68d76e2.png">

Here's what it looks like with the title reverted back to `--dark-200`

<img width="1725" alt="Screen Shot 2022-07-19 at 4 38 48 PM" src="https://user-images.githubusercontent.com/35148959/179853432-7476f2b7-768a-45cf-88d4-60944cff4748.png">